src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -35,22 +35,22 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
   case class SFNRequest(stateMachineArn: String, name: String, input: String)
 
   val reference = "TEST-REFERENCE"
-  val uuids: List[String] = List(
-    "c7e6b27f-5778-4da8-9b83-1b64bbccbd03",
-    "61ac0166-ccdf-48c4-800f-29e5fba2efda",
-    "4e6bac50-d80a-4c68-bd92-772ac9701f14",
-    "c2e7866e-5e94-4b4e-a49f-043ad937c18a",
-    "27a9a6bb-a023-4cab-8592-39b44761a30a"
+  val uuids: List[(String, String)] = List(
+    ("c7e6b27f-5778-4da8-9b83-1b64bbccbd03", "71"),
+    ("61ac0166-ccdf-48c4-800f-29e5fba2efda", "81"),
+    ("4e6bac50-d80a-4c68-bd92-772ac9701f14", "91"),
+    ("c2e7866e-5e94-4b4e-a49f-043ad937c18a", "A1"),
+    ("27a9a6bb-a023-4cab-8592-39b44761a30a", "B1")
   )
 
-  val metadataFiles: List[String] = List(
-    "asset-metadata.csv",
-    "bagit.txt",
-    "bag-info.txt",
-    "file-metadata.csv",
-    "folder-metadata.csv",
-    "manifest-sha256.txt",
-    "tagmanifest-sha256.txt"
+  val metadataFiles: List[(String, String)] = List(
```

Comment:
Do you think it's worth adding what that second string is, in the val name "metadataFilesAndX", because it's hard to tell what it is?

Reply:
I've changed this to `metadataFilesAndChecksums`